docker-compose.yml

```diff
@@ -253,6 +256,10 @@ services:
       retries: 5
     networks:
       - pixl-net
+    # needed for testing under GHA (linux), so this container
+    # can reach the test FTP server running on the docker host
+    extra_hosts:
```

Comment:
If this is only needed for testing, I wonder if its worth making the build context for the test docker compose the same and then adding this only when running the test. If you think thats worth it, then create an issue but not blocking this PR

Reply:
I did want to do this but didn't work out how.